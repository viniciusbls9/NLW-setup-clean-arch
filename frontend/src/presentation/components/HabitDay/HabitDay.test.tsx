import { describe, test, expect } from 'vitest';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { HabitDay } from '.';

const props = {
  completed: 4,
  amount: 5
};

describe('HabitDay Component', () => {
  test('Should render correctly HabitDay component', () => {
    const { container } = render(<HabitDay {...props} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
