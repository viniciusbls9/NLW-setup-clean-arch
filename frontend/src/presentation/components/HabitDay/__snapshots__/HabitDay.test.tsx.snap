// Vitest Snapshot v1

exports[`HabitDay Component > Should render correctly HabitDay component 1`] = `
<button
  aria-controls="radix-:r0:"
  aria-expanded="false"
  aria-haspopup="dialog"
  class="w-10 h-10 bg-zinc-900 border-2 border-zinc-800 rounded-lg"
  data-state="closed"
  type="button"
/>
`;
